src/eradiate/scenes/bsdfs/_rtls.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import annotations
+
+import attrs
+import pinttr
+import mitsuba as mi
+
+from ._core import BSDF
+from ..core import traverse
+from ..spectra import Spectrum, spectrum_factory
+from ... import validators
+from ...units import unit_context_config as ucc
+from ...attrs import documented, parse_docs
+from ...kernel import TypeIdLookupStrategy, UpdateParameter
+
+
+@parse_docs
+@attrs.define(eq=False, slots=False)
+class RTLSBSDF(BSDF):
+    """
+    RTLS BSDF [``rtls``].
+
+
+
+    See Also
+    --------
+    :ref:`plugin-bsdf-rtls`
+
+    Implements the Ross-Thick, Li-Sparse (RTLS) BRDF
+    :cite:`BU-MODISBRDFAlbedoProductATBD1999`.
+
+    The RTLS class exposes the 3 parameters :math:`f_{iso}`, :math:`f_{vol}` and
+    :math:`f_{geo}` configuring the linear combination the wreights of the
+    isometric, volumetric and geometric kernel values in the BSDF definition
+    respectively.
+    """
+
+    f_iso: Spectrum = documented(
+        attrs.field(
+            default=0.209741,
+            converter=spectrum_factory.converter("dimensionless"),
+            validator=[
+                attrs.validators.instance_of(Spectrum),
+                validators.has_quantity("dimensionless"),
+            ],
+        ),
+        doc="Isotropic scattering kernel parameter. "
+        r"Defaults to the Mitsuba plugin default (equal to :math:`f_{iso}`).",
```

Comment:
Am I understand this correctly? From what I read here, the default is simply 0.209741, not the Mitsuba plugin default.

Reply:
0.209741 [is the plugin default](https://github.com/eradiate/mitsuba3/blob/373e367b2c5720c9d4f3a8e83477b4cbc838cc2d/src/eradiate_plugins/bsdfs/rtls.cpp#L23). I thought implementing conditionals here would be more tedious than simply copying a value that end user will override anyway. I can look into something nicer

Reply:
I'm fine using a hard-coded default as well. So you have two possible solutions: either change the behaviour (default to `None` and specify the plugin init value and parameters only if a non-default value is passed) or change the docs to mention explicitly the defualt value.

Reply:
Let me know what you think

Reply:
Looks good to me now :+1: